src/composables/notifications.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  computed, onBeforeUnmount, onMounted, ref,
+} from '@vue/composition-api';
+import {
+  IAccount, IChainNames,
+  INetwork,
+  INotification,
+  INotificationSetting,
+  NotificationStatus,
+} from '../types';
+import { useGetter, useState } from './vuex';
+import {
+  NOTIFICATION_STATUS_CREATED,
+  NOTIFICATION_STATUS_READ,
+  NOTIFICATION_TYPE_WALLET,
+  executeAndSetInterval,
+  watchUntilTruthy, fetchJson, postJson, NOTIFICATION_ENTITY_TYPE_TIP, AGGREGATOR_URL,
```

Comment:
I suggest to split this line same as the previous ones with the constants on top.